README.md

```diff
@@ -76,59 +88,64 @@ Here are the steps needed to run Terraform along with examples of each:
 
 `terraform apply -var-file="secret.tfvars"`
 
-Each of these commands includes the `secret.tfvars` containing the sensitive variables needed to connect to the different resources as described in the previous section.
+Each of these commands includes the `secret.tfvars` containing the sensitive variables that are needed to connect to the different resources as described in the previous section.
 
 ## Results
 
 ### Virtual Machines
 
-You will see 5 virtual machines created with static IP addresses in vSphere.
+You see five virtual machines that are created with static IP addresses in vSphere.
 
-![Virtual Machines](images/vsphere-virtual-machines.png)
+<img src="images/vsphere-virtual-machines.png" alt="Virtual Machines">
```

Comment:
edit to the alt text: "Virtual Machines screenshot of the vSphere client"

Reply:
Take a look at https://supercooldesign.co.uk/blog/how-to-write-good-alt-text for some alt text ideas, it's possible my suggestions are too many characters.